distribution.pm

```diff
@@ -130,15 +131,16 @@ sub script_run {
     my ($self, $cmd) = splice(@_, 0, 2);
     my %args = testapi::compat_args(
         {
-            timeout => $bmwqemu::default_timeout,
-            output  => '',
-            quiet   => undef
+            timeout      => $bmwqemu::default_timeout,
+            output       => '',
+            quiet        => undef,
+            max_interval => undef
         }, ['timeout'], @_);
 
     if (testapi::is_serial_terminal) {
         testapi::wait_serial($self->{serial_term_prompt}, no_regex => 1, quiet => $args{quiet});
     }
-    testapi::type_string "$cmd";
+    testapi::type_string("$cmd", max_interval => $args{max_interval});
```

Comment:
please also see my comment further down about passing `%args` unaltered. Maybe just `testapi::type_string $cmd, %args` is enough

Reply:
indeed

Reply:
I think we would mix up the parameters for script_run with the ones we want to parse further down? e.g. timeout should not be passed here if we handle it in the line afterwards

Reply:
If I understand the test output correctly we caused exactly that problem, as we pass quiet further down than it should be and therefore the default is wrong

Reply:
@DrMullings actually @okurz is right, passing it should help, however the quet argument can be deleted from the hash before passing it